### PR TITLE
{system}[GCC/8.3.0] CRIU v3.13, future v0.16.0, protobuf-c v1.3.3 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by jordiblasco**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jordiblasco** [comment]
Hi,

Please, provide advice about the sanity_pip_check. I suspect that it is complaining about the empty file array in sanity_check_paths. I used the following recipe as a template:

https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/f/future/future-0.16.0-foss-2018b-Python-3.6.6.eb

Thanks!

**Comment by jordiblasco** [comment]
Hi @boegel!

Do you have any updates about this issue?

Thanks!

Regards,

Jordi

**Comment by jordiblasco** [comment]
Updated the headers just to trigger the unit tests

**Comment by boegel** [comment]
@jordiblasco Sorry for the lack of a response here!

Can you take a look at the failing tests? Hopefully the error messages are clear enough...

File: easybuild/easyconfigs/c/CRIU/CRIU-3.13-GCC-8.3.0-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('protobuf', '3.10.0'),
+    ('protobuf-c', '1.3.3'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('future', '0.16.0', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/c/CRIU/CRIU-3.13-GCC-8.3.0-Python-3.7.4.eb:32]: `future` is already included in the `Python` 3.7.4 easyconfig, so this extra dependency is probably overkill?

**Comment by boegel** [comment]
I had an interest in getting this ready to merge, but I hit a wall due to missing OS dependencies.

I did clean this up a little bit though, w.r.t. the hardcoding in `dependencies`.
As far as I understand, a compiler-only toolchain is also sufficient for CRIU.

There's probably a bit more work needed to get this in, like dropping the `future` dependency, which is overkill, I think?

I also wasn't able to verify whether the sanity check commands I added for CRIU are actually working...

**Pull request closed without merging**